{% extends 'layout.html' %}
{% block body %}
<h3>How to use gutSMASH</h3>
<div>
  <div class="well well-small">
    Check our publication for more information:
    <br />
    <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7874-8_15"> 
    </a>
  </div>
  <div id="intro" style="margin: 1em;">
    <h4>Introduction</h4>
    <p>
      Anaerobic bacteria in the gut and they are responsible for the synthesis and transformation of diverse molecules involved in host-microbe and microbe-microbe interactions, that ultimately derive host phenotypes. The pathways encoding the production of these molecules belong to a specialized primary metabolism and are often physically clustered in the genome, in regions also known as metabolic gene clusters (MGCs)
    </p>
    <p>
      gutSMASH is a tool that has been developed to fulfill the need of systematically evaluate the metabolic potential of these bacteria by predicting both known and novel anaerobic MGCs from the gut microbiome.
      The gutSMASH detection rules have been validated using a training set presented in our main publication
    </p>
    <p>
      Altogether, this new software provides a comprehensive toolkit to functionally characterize anaerobic bacterial genomes by not only predicting MGCs of known functions but also novel MGCs that may represent good candidates for further experimental characterization.
    </p>
  </div>
  <div id="input" style="margin: 1em;">
    <h4>gutSMASH input parameters</h4>
    <p>
      The ideal input for gutSMASH is an annotated nucleotide file in Genbank format or EMBL format.
      You can either upload a GenBank/EMBL file manually, or simply enter the GenBank/RefSeq accession number of your sequence for gutSMASH to upload it.
      Alternatively, you can provide a FASTA file containing one or more sequences.
      If possible, you should also supply a separate GFF3-formatted file containing the annotation information for all said sequences.
      Otherwise, gutSMASH will generate a preliminary annotation using GlimmerHMM, and use that to run the rest of the analysis.
    </p>
    <p>
      Input files should be properly formatted.
      If you are creating your GBK/EMBL/FASTA file manually, be sure to do so in a plain text editor like Notepad or Emacs, and saving your files as "All files (*.*)", ending with the correct extension (for example ".fasta", ".gbk", or ".embl").
    </p>
    <p>
      There are several optional analyses that may or may not be run on your sequence.
      Highly recommended is the ClusterBlast Comparative Analysis, which runs Diamond using each amino acid sequence from a detected gene cluster as a query on a large database of predicted protein sequences from secondary metabolite biosynthetic gene clusters, and pools the results to identify the gene clusters that are most homologous to the gene cluster that was detected in your query nucleotide sequence.
    </p>
  </div>
  <div id="output" style="margin: 1em;">
    <h4>gutSMASH output</h4>
    <p>
      The output of the gutSMASH analysis pipeline is organized in an interactive HTML page with SVG graphics, and different parts of the analysis are displayed in different panels for every gene cluster.
    </p>
    <img src="{{ url_for('static', filename='images/image002.png') }}">
    <p>
      Initially, a list of identified clusters is displayed in the results page.
      A gene cluster can be selected for viewing by clicking its number (gene clusters are numbered in the order in which they appear on the input nucleotide sequence) in the "Overview" panel just below the top banner or by clicking on the "Region XX" boxes.
      A click on "Overview" brings you back to the overview list.
      <br />
      Gene cluster buttons are color-coded by predicted hybrids or non-hyrbrids.
    </p>
    <img src="{{ url_for('static', filename='images/image004.png') }}">
    <p>
      In the upper panel, "Gene cluster description", information is given about each gene cluster that was detected.
      In the upper line the length and location of the gene cluster are displayed.
      Underneath this title line, all genes present in a detected gene cluster are outlined.
    </p>
    <p>
      Genes are color-coded by their assigned enzymatic protein domain profile, which will be listed in the legend.
      Additionally, genes will also be categorized by its predicted function, which will be displayed as different color borders.
    </p>
    <p>
      Hovering over a gene with the mouse will prompt the gene name to be displayed above the gene.
      Clicking the gene will provide more information on the gene: its annotation, its sequence similarity with other genes in the cluster, its location, and cross-links specific to that gene.
    </p>
      You can choose to show either expression fluctuation (the rate of which expression level of a gene changes between samples), color-coded from white to black; or expression intensity (expression level of a gene related to the sample value distribution), color coded from yellow to red.
    <img src="{{ url_for('static', filename='images/image006.png') }}">
  
    <img src="{{ url_for('static', filename='images/image008.png') }}">
    <p>
      If you enabled Gene Cluster Comparative Analysis (ClusterBlast), this panel will display the top ten plant gene clusters from precomputed data that are most similar to a detected gene cluster, visually aligned to it.
      The drop-down selection menu can be used to browse through the gene clusters.
      Genes with the same colour are putative homologs based on significant Blast (Diamond) hits between them.
    </p>
    <img src="{{ url_for('static', filename='images/image020.png') }}">
    <p>
      In the upper right, a small list of buttons offers further functionality.
      The house-shaped button will get you back to the <a href="http://plantismash.secondarymetabolites.org">gutSMASH start page</a>.
      The question-mark button will get you to this help page.
      The exclamation-mark button leads to a <a href="http://plantismash.secondarymetabolites.org/about.html">page explaining about gutSMASH</a>.
      The downward-pointing arrow will open a menu offering to download the complete set of results from the antiSMASH run, a summary Excel file and to the summary EMBL/GenBank output file.
      The EMBL/GenBank file can be viewed in a genome browser such as <a href="http://www.sanger.ac.uk/science/tools/artemis">Artemis</a>.
    </p>
  </div>
  <div id="faq">
    <h4>Frequently Asked Questions</h4>
    <ol>
      <li>
        <div class="question">Can I run gutSMASH locally as a stand-alone program?</div>
        Yes.
        A stand-alone version of gutSMASH is available from the <a href="http://plantismash.secondarymetabolites.org/download.html">download section of this website</a>.
        GutSMASH natively run on python 3 under GNU/Linux, but will also run on other environment as well (OSX / Windows), given the dependency requirements are fulfilled.
      </li>
      <li>
        <div class="question">Why doesn't gutSMASH detect my gene cluster?</div>
        Some gene clusters, such as fatty acid or cofactor biosynthesis gene
        clusters are not identified by antiSMASH, as they are categorized as <a
          href="http://en.wikipedia.org/wiki/Primary_metabolism" target="_blank">primary
          metabolism</a> instead of <a
          href="http://en.wikipedia.org/wiki/Secondary_metabolism" target="_blank">secondary
          metabolism</a>. If you are aware of a true secondary metabolite
        biosynthesis gene cluster that escapes detection by antiSMASH, please
        contact us, and we will add the models necessary to detect it.
      </li>
      <li>
        <div class="question">Can I also submit an unannotated genome sequence in FASTA format?</div>
        Yes. We offer integrated preliminary gene prediction by GlimmerHMM based on a FASTA input.
      </li>
      <li>
        <div class="question">What is the privacy policy of antiSMASH concerning my sequence data?</div>
        We try to keep this site and the data that it analyzes as safe and secure as possible. Your output files will be deleted from our server within one month. However, sending your data to the web site is at your own risk. If you are concerned about the sensitivity of your data, please use the stand-alone version of our tool.
      </li>
    </ol>
  </div>
  <div id="secmettypes">
    <h4>Secondary metabolite types detected</h4>
    <dl>
      <dt id="alkaloid">alkaloid</dt>
      <dd>Putative alkaloid biosynthetic cluster. Characterized by the presence of Bet_v_1/Cu_amine_oxid/Str_synth enzymes.</dd>
      <dt id="lignan">lignan</dt>
      <dd>Putative lignan biosynthetic cluster. Characterized by the presence of Dirigent enzyme.</dd>
      <dt id="polyketide">alkaloid</dt>
      <dd>Putative polyketide biosynthetic cluster. Characterized by the presence of Dirigent enzymes.</dd>
      <dt id="saccharide">saccharide</dt>
      <dd>Putative saccharide biosynthetic cluster. Characterized by the presence of Chal_sti_synt enzymes.</dd>
      <dt id="terpene">terpene</dt>
      <dd>Putative terpene biosynthetic cluster. Characterized by the presence of Terpene_synth/Prenyltrans/SQHop_cyclase/PRISE enzymes.</dd>
      <dt id="putative">putative</dt>
      <dd>Putative / uncategorized biosynthetic cluster</dd>
      <dt id="hybrid">{A} â€“ {B}</dt>
      <dd>Biosynthetic cluster containing signature traces from both A and B secondary metabolite type.</dd>
    </dl>
  </div>
</div>
{% endblock %}
